pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -541,7 +628,8 @@
     "runtime.win7-x64.runtime.native.System.IO.Compression": {
       "StableVersions": [
         "4.0.1"
-      ]
+      ],
+      "BaselineVersion": "4.3.0"
     },
     "runtime.win7-x86.runtime.native.System.Data.SqlClient.sni": {
```

Comment:
@saurabh500 Are there corresponding changes in managed code for the supporting SNI native code that was checked into TFS? If so, is that managed SNI code in CoreFX repo or in TFS as well?

@weshaggard What are your thoughts on this?